src/components/hv-web-view/index.tsx

```diff
@@ -38,10 +38,12 @@ export default class HvWebView extends PureComponent<HvComponentProps> {
       this.props.options,
     );
     const color = props['activity-indicator-color'] || '#8d9494';
+    const allowsInlineMediaPlayback = props['allows-inline-media-playback'];
```

Comment:
I think we need to cast this to a boolean and only set when attribute exist:
```suggestion
    const allowsInlineMediaPlayback = props['allows-inline-media-playback'] ? props['allows-inline-media-playback'] === 'true' : undefined;
```